install/jpa/docs/assertions/spec/spec-assertions-3_0.html

```diff
@@ -4668,11 +4668,11 @@ <h2> Jakarta Persistence - 3.0<br>
 </font></TD><TD align="center" valign="center"><font size="1PT">false</font></TD><TD align="center" valign="center"><font size="1PT">technology</font></TD><TD align="center" valign="center"><font size="1PT">active</font></TD><TD align="center" valign="center"><font size="1PT">true</font></TD>
 </TR>
 <TR>
-<TD align="center" valign="center"><a name="PERSISTENCE:SPEC:1318"></a><font size="1PT">PERSISTENCE:SPEC:1318</font></TD><TD align="center" valign="center"><font size="1PT">3</font></TD><TD align="center" valign="center"><font size="1PT">5.1</font></TD><TD align="left" valign="center"><font size="1PT">Entity listener classes in Java EE environments support
+<TD align="center" valign="center"><a name="PERSISTENCE:SPEC:1318"></a><font size="1PT">PERSISTENCE:SPEC:1318</font></TD><TD align="center" valign="center"><font size="1PT">3</font></TD><TD align="center" valign="center"><font size="1PT">5.1</font></TD><TD align="left" valign="center"><font size="1PT">Entity listener classes in Jakarta EE environments support
                 dependency injection through the
-                Contexts and
-                Dependency Injection API (CDI) when the CDI is enabled. An entity listener
-                class that makes use of CDI injection may also define lifecycle
+                Jakarta Contexts and
+                Dependency Injection API (JCDI) when the JCDI is enabled. An entity listener
```

Comment:
Is JCDI referenced elsewhere or are we starting a new trend here?  :-)  

From google searching for JCDI I did find two matches that included `Java Contexts and Dependency Injection (JCDI)`

Reply:
Is CDI acronym allowed? Is it clear that all occurrences of CDI are referring to "Jakarta Contexts and Dependency Injection and not "JSR" "Contexts and Dependency Injection"? How should I properly define "new" acronym so I can use it later in the text if not by "<Some Long Name> (SLN)" way? Or do you want me to replace all "CDI"s by "Jakarta Contexts and Dependency Injection"?

Reply:
I am not a lawyer but CDI is not mentioned on https://jakarta.ee/legal/acronym_guidelines, so I think we can leave it as `CDI`.

Reply:
Neither am I. If you think it's OK, then just leave it as it was...